site/_layouts/documentation.html

```diff
@@ -43,10 +43,10 @@
 
             <!-- If the current version has a explicitly versioned sidebar for the major version -->
             {% if site.doc_versions contains current_version %}
-                {% capture sidebar %}documentation-sidebar/{{major_version}}/sidebar.html{% endcapture %}
+                {% capture sidebar %}{{page.category}}-sidebar/{{major_version}}/sidebar.html{% endcapture %}
```

Comment:
This dynamically loads a different sidebar based on a page's category.

Reply:
So if a page doesn't set `getting-started` as a category, it defaults to `documentation`, which will load `documentation-sidebar/<version>/sidebar.html`.